Safewaan/HonkedIn
# Style Forums and My Forums Page

## Trello ticket link

[Developer Task: Implement user feedback for UI/UX of Forums Page](https://trello.com/c/GHAheSjq/135-developer-task-implement-user-feedback-for-ui-ux-of-forums-page)



## Implementation description
* Styled Forums and My Forums Page
* Styled all dialogue options (Archive Forum, Edit Forum)



## Steps to test
1. Verify all dialogue options are working
2. Verify there are no styling inconsistencies
3. Verify correct errors are thrown in error cases


## What should reviewers focus on?
*
